profiles/adb.jsonld

```diff
@@ -90,9 +80,7 @@
         "prefLabel": {
         "en": "coached"
       },
-      "related": [ {
-        "id": "https://w3id.org/xapi/adb/verbs/requested"
-      } ]
+      "related": ["https://w3id.org/xapi/adb/verbs/requested"]
```

Comment:
Coached and requested are not related by having similar meanings, which is what the `related` property means, so this should be removed.